projects/aca-content/src/lib/components/toolbar/toggle-edit-offline/toggle-edit-offline.component.ts

```diff
@@ -117,13 +118,15 @@ export class ToggleEditOfflineComponent implements OnInit {
   }
 
   lockNode(nodeId: string) {
+    this.nodeActionsService.setNodeLocked(true);
     return this.nodesApi.lockNode(nodeId, {
       type: 'ALLOW_OWNER_CHANGES',
       lifetime: 'PERSISTENT'
     });
   }
 
   unlockNode(nodeId: string) {
+    this.nodeActionsService.setNodeLocked(false);
```

Comment:
should be called only when "unlockNode" is complete, use "subscribe" or "then"

Reply:
https://github.com/Alfresco/alfresco-content-app/pull/3414#discussion_r1318531727